praw/models/reddit/mixins/gildable.py

```diff
@@ -1,26 +1,111 @@
 """Provide the GildableMixin class."""
+from warnings import warn
+
 from ....const import API_PATH
 
 
 class GildableMixin:
     """Interface for classes that can be gilded."""
 
     def gild(self):
-        """Gild the author of the item.
+        """Alias for :meth:`.award` to maintain backwards compatibility."""
+        warn(
+            "`.gild` has be renamed to `.award`.",
```

Comment:
This should probably be ```"`.gild` was renamed to `.award`.",``` or something to that effect

Reply:
I think present tense makes more sense here.